gcp/rakefiles/sh_filter.rb

```diff
@@ -1,13 +1,14 @@
 # Obfuscate sensitive data in the output of a command, relying on @secrets
 def sh_filter(*cmd)
-  if @secrets
-    IO.popen(*cmd).each do |out|
+  IO.popen(*cmd).each do |out|
+    # After switching back to helm-release, plaintext certificates and keys may be present in output
+    out.gsub!(/-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----\\n/m, "<SENSITIVE>")
```

Comment:
I would leave the BEGIN/END lines for clarity, and only replace the contents of the cert with `<SENSITIVE>`.

Reply:
I think it is clear enough from logs that cert body is being replaced:
```module.system_tiller.local_file.tiller_cert: Creating...
  content:  "" => "<SENSITIVE>"
  filename: "" => "/project/live/dev/secrets/kube-system/helm-tls/tiller.cert.pem"```

Reply:
Ah, ok.